src/components/Button/Button.scss

```diff
@@ -13,13 +13,14 @@
 
   &--grey {
     background-color: $clr-accent-grey;
-    color: white;
+    color: $clr-white;
     border: none;
   }
 
   &--red {
     background-color: $clr-accent-red;
     border: none;
+    color: $clr-white;
```

Comment:
why is this the same variable as the grey colour above?

Reply:
<img width="303" alt="image" src="https://github.com/nology-tech/verde-client-project/assets/148501184/9ba166e7-d4de-4dbe-b700-5707074726c9">
The font was black not white like on figma (the picture attached), that's why this line was added. I think the settings page is the first page that makes use of the red variant of the button.